software/search.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+---
+
+{% capture my_include %}{% include software-search-intro.md %}{% endcapture %}
+{{ my_include | markdownify }}
+
+<section aria-label="Big search component">
+  <form id="software-search" class="usa-search usa-search--big" role="search">
+    <label class="usa-sr-only" for="search-field-big">Search</label>
+    <input class="usa-input" type="search" name="search" value="office" />
```

Comment:
Is the default value `"office"` intentional? It adds a bit of extra effort when landing at the page, since I first must clear the value before searching for what I want.

Reply:
Fair. The thought was to kinda demonstrate what the page does via an already-searched example.

Reply:
Removed it.